foreman/data_refinery_foreman/foreman/main.py

```diff
@@ -250,7 +265,7 @@ def prioritize_jobs_by_accession(jobs: List, accession_list: List[str]) -> List:
                         is_prioritized_job = True
                         break
         except:
-            logger.exception("Exception caught while prioritizing zebrafish jobs!")
+            logger.exception("Exception caught while prioritizing zebrafish jobs!", job=job)
```

Comment:
This has the same error message as the one above, which can make debugging harder. Make these distinct